core/services/src/main/resources/changelog/attachments-context-rdbms.db.changelog.xml

```diff
@@ -45,4 +45,23 @@
     </modifySql>
   </changeSet>
 
+  <changeSet author="attachments" id="1.0.0-2">
+    <customChange class="org.exoplatform.services.attachments.liquibase.DropDuplicateAndNullAttachmentIds">
+      <param name="tableName">EXO_ATTACHMENTS_CONTEXT</param>
+    </customChange>
+  </changeSet>
```

Comment:
Needs to be teste with Ops before putting in production. @hbenali @aelbenney I let you organize a test on Tribe QA server & @hghorbel to add a new Us on board for it